include/graphics/window.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <GLFW/glfw3.h>
```

Comment:
I think that you can use `struct GLFWwindow` here, to avoid `#include`ing as many files in headers.